source: https://gitlab.com/wavexx/acpilight/commit/f54865ed9a11eedaeffa71af320a3bf36c89f15d.patch
This patch is taken from upstream and should therefore be obsolete in the
next release > 1.1

From f54865ed9a11eedaeffa71af320a3bf36c89f15d Mon Sep 17 00:00:00 2001
From: Yuri D'Elia <wavexx@thregr.org>
Date: Fri, 6 Jul 2018 12:34:34 +0200
Subject: [PATCH] Fix udev rules for "leds" (kbd backlight)

---
 90-backlight.rules | 4 ++--
 README.rst         | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/90-backlight.rules b/90-backlight.rules
index d412314..2612365 100644
--- 90-backlight.rules
+++ 90-backlight.rules
@@ -3,5 +3,5 @@ SUBSYSTEM=="backlight", ACTION=="add", \
   RUN+="/bin/chgrp video /sys/class/backlight/%k/brightness", \
   RUN+="/bin/chmod g+w /sys/class/backlight/%k/brightness"
 SUBSYSTEM=="leds", ACTION=="add", KERNEL=="*::kbd_backlight", \
-  RUN+="/bin/chgrp video /sys/class/backlight/%k/brightness", \
-  RUN+="/bin/chmod g+w /sys/class/backlight/%k/brightness"
+  RUN+="/bin/chgrp video /sys/class/leds/%k/brightness", \
+  RUN+="/bin/chmod g+w /sys/class/leds/%k/brightness"
diff --git a/README.rst b/README.rst
index b043d44..61bb835 100644
--- a/README.rst
+++ b/README.rst
@@ -62,8 +62,8 @@ The following rules allow users in the video group to set the keyboard
 backlight as well::
 
   SUBSYSTEM=="leds", ACTION=="add", KERNEL=="*::kbd_backlight", \
-    RUN+="/bin/chgrp video /sys/class/backlight/%k/brightness", \
-    RUN+="/bin/chmod g+w /sys/class/backlight/%k/brightness"
+    RUN+="/bin/chgrp video /sys/class/leds/%k/brightness", \
+    RUN+="/bin/chmod g+w /sys/class/leds/%k/brightness"
 
 Please report if other laptops have working interfaces to control the keyboard
 backlight.
-- 
2.18.1


